packages/core/r4b/data-types/boolean.ts

```diff
@@ -0,0 +1,88 @@
+import { merge } from "lodash";
+/**
+ * A boolean
+ *
+ * @see https://hl7.org/fhir/datatypes.html#boolean
+ */
+
+export interface FhirBooleanFormatOptions {
+  labels?: {
+    true?: string;
+    false?: string;
+    null?: string;
+    undefined?: string;
```

Comment:
Generally, `null` and `undefined` are functionally equivalent.
So we could group them as `nil` to make it easier?

```suggestion
    nil?: string;
```